targets/clojurescript/index.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var evalCLJS = require('./cljs');
+
+module.exports = function cljsToJs(resolve, reject, data) {
+  try {
+    evalCLJS(data.source, function(err, result) {
+      resolve({
+        errors: err || null,
+        result: result || null
+      });
+    });
+  } catch (e) {
+    console.log('failed ClojureScript');
+    resolve({
+      errors: [e],
```

Comment:
Honestly I've no idea, can't force it to throw an error 😅. In fact this `try...catch` will never be given an error, because internally CLJS has wrapped compilation operation into another `try...catch` and all errors are being passed down to the callback function. Also there's one more `try...catch` which wraps evaluation operation and also all errors are going into the callback function.